airbyte-integrations/connectors/source-amazon-seller-partner/source_amazon_seller_partner/schemas/OrderItems.json

```diff
@@ -0,0 +1,269 @@
+{
+    "title": "Order Items",
+    "description": "All order items that were updated after a specified date",
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "additionalProperties": true,
+    "properties": {
+        "AmazonOrderId": {
+            "type": "string"
```

Comment:
Can you add the pattern `["null", "data type"]`? It isn't guarantee to always receive data for each field.

Reply:
ok